alchemist-loading/src/test/java/it/unibo/alchemist/test/PreventRegressions.java

```diff
@@ -31,11 +28,16 @@
      */
     @Test
     void testLoadCustomExport() {
-        final List<Extractor> extractors = LoadAlchemist.from(ResourceLoader.getResource("testCustomExport.yml"))
+        LoadAlchemist
+            .from(ResourceLoader.getResource("testCustomExport.yml"))
             .getDefault()
-            .getDataExtractors();
-        assertEquals(1, extractors.size());
-        assertEquals(MeanSquaredError.class, extractors.get(0).getClass());
+            .getExporters()
+            .stream()
+            .findFirst()
+            .ifPresent(exporter -> {
+                assertEquals(1, exporter.getDataExtractors().size());
+                assertEquals(MeanSquaredError.class, exporter.getDataExtractors().get(0).getClass());
+            });
```

Comment:
This test does not fail if none is present. You must assert that a result is present.